mods/ra/maps/allies-10b/allies10b.lua

```diff
@@ -0,0 +1,323 @@
+--[[
+   Copyright 2007-2023 The OpenRA Developers (see AUTHORS)
```

Comment:
```suggestion
   Copyright (c) The OpenRA Developers and Contributors
```

We changed to a different copyright header

Reply:
Oh that's way smarter. Updated.